README.md

```diff
@@ -2,7 +2,7 @@ pylint-django
 =============
 
 [![Build Status](https://travis-ci.org/PyCQA/pylint-django.svg?branch=master)](https://travis-ci.org/PyCQA/pylint-django)
-[![Code Quality](https://landscape.io/github/landscapeio/pylint-django/master/landscape.png)](https://landscape.io/github/landscapeio/pylint-django)
+[![Code Quality](https://landscape.io/github/PyCQA/pylint-django/master/landscape.png)](https://landscape.io/github/PyCQA/pylint-django)
```

Comment:
This is a 404 URL, I don't think you should change it. Probably better to remove this badge.

Reply:
I believe we should have this repository checks by Landscape. Currently, Landscape has some issues, @carlio may know, but as soon as it works fine again the health value carries a valuable feedback.

Provided you have (admin) permissions on this repository you need to refresh the connection with GitHub on Landscape, then the 404 should go away. Then, you should see a page saying, "This repository is not yet checked", which allows you to activate the health checking.

Reply:
Yeah I'm working on that, it's a bit brokkken for now. However it would only make sense to update the link once the target of the link is working.